button.js

```diff
@@ -3,28 +3,25 @@ function changeColor() {
   // Use the Mozilla Developer Network resource to find the right way to change
   // the background for this div 
   /* Your Code Goes Here */
+
+  document.getElementById("changeable").style.backgroundColor = "yellowGreen"
+
+  
 }
 
 function replaceWord() {
+  var ran = Math.floor(Math.random( )*100);
   var wordable = document.getElementById('wordable');
-
-  fetch("https://wordsapiv1.p.rapidapi.com/words/?random=true", {
-    method: "GET",
-    headers: {
-      "x-rapidapi-key": "your-api-key-here",
-      "x-rapidapi-host": "api-host-here",
-    },
+  var word = "";
+  //var blueDiv = document.getElementById("changeable");
+  fetch('https://jsonplaceholder.typicode.com/posts/'+ran)
+      .then(data => {return data.json();})
+      .then(word => {word =>word.title;console.log(word.title);
```

Comment:
I can see that this will work, but I'm curious about why you've included a second function inside this function body. what does the `word => word.title;` function acocmplish?

Reply:
That's correct that was not needed, forgot to remove it, I was just testing out some results on console.